<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QLabel" name="lblSong" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>193</red>
            <green>193</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>155</red>
            <green>155</green>
            <blue>226</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>58</red>
            <green>58</green>
            <blue>99</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>78</red>
            <green>78</green>
            <blue>132</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>186</red>
            <green>186</green>
            <blue>226</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>193</red>
            <green>193</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>155</red>
            <green>155</green>
            <blue>226</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>58</red>
            <green>58</green>
            <blue>99</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>78</red>
            <green>78</green>
            <blue>132</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>186</red>
            <green>186</green>
            <blue>226</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>58</red>
            <green>58</green>
            <blue>99</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>193</red>
            <green>193</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>155</red>
            <green>155</green>
            <blue>226</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>58</red>
            <green>58</green>
            <blue>99</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>78</red>
            <green>78</green>
            <blue>132</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>58</red>
            <green>58</green>
            <blue>99</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>58</red>
            <green>58</green>
            <blue>99</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>117</red>
            <green>117</green>
            <blue>198</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font" >
       <font>
        <family>Verdana</family>
        <pointsize>21</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="midLineWidth" >
       <number>3</number>
      </property>
      <property name="text" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Verdana'; font-size:21pt; font-weight:600; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:400;">Current Song&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Queue</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QSplitter" name="splitter_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Search</string>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="leftMargin" >
               <number>0</number>
              </property>
              <property name="topMargin" >
               <number>0</number>
              </property>
              <property name="rightMargin" >
               <number>0</number>
              </property>
              <property name="bottomMargin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>Find</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="edtFind" />
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSplitter" name="splitter_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QWidget" name="layoutWidget" >
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>6</number>
                </property>
                <property name="leftMargin" >
                 <number>0</number>
                </property>
                <property name="topMargin" >
                 <number>0</number>
                </property>
                <property name="rightMargin" >
                 <number>0</number>
                </property>
                <property name="bottomMargin" >
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2" >
                  <property name="text" >
                   <string>Artists</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="lbArtists" />
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget" >
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_8" >
                  <property name="text" >
                   <string>Albums</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="lbAlbums" />
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget" >
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>6</number>
                </property>
                <property name="leftMargin" >
                 <number>0</number>
                </property>
                <property name="topMargin" >
                 <number>0</number>
                </property>
                <property name="rightMargin" >
                 <number>0</number>
                </property>
                <property name="bottomMargin" >
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3" >
                  <property name="text" >
                   <string>Songs</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="lbSongs" />
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="leftMargin" >
               <number>0</number>
              </property>
              <property name="topMargin" >
               <number>0</number>
              </property>
              <property name="rightMargin" >
               <number>0</number>
              </property>
              <property name="bottomMargin" >
               <number>0</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnQueue" >
                <property name="text" >
                 <string>Add to queue</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QSplitter" name="splitter" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="layoutWidget" >
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="leftMargin" >
              <number>0</number>
             </property>
             <property name="topMargin" >
              <number>0</number>
             </property>
             <property name="rightMargin" >
              <number>0</number>
             </property>
             <property name="bottomMargin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5" >
               <property name="text" >
                <string>History</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="lbHistory" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget" >
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="leftMargin" >
              <number>0</number>
             </property>
             <property name="topMargin" >
              <number>0</number>
             </property>
             <property name="rightMargin" >
              <number>0</number>
             </property>
             <property name="bottomMargin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4" >
               <property name="text" >
                <string>Current queue</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="lbQueue" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Dynamic Playlist</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="cbEnabled" >
          <property name="text" >
           <string>Enable dynamic playlist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget" >
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Artists</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="lbArtistsDPL" >
              <property name="dragEnabled" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget" >
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Dynamic playlist includes all songs where:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="edtDPL" />
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveDPL" >
            <property name="text" >
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtFind</tabstop>
  <tabstop>lbArtists</tabstop>
  <tabstop>lbAlbums</tabstop>
  <tabstop>lbSongs</tabstop>
  <tabstop>btnQueue</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lbHistory</tabstop>
  <tabstop>lbQueue</tabstop>
  <tabstop>cbEnabled</tabstop>
  <tabstop>lbArtistsDPL</tabstop>
  <tabstop>edtDPL</tabstop>
  <tabstop>btnSaveDPL</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
